syntax = "proto3";
package cl_rcon;
option optimize_for = LITE_RUNTIME;

enum request_t
{
	SERVERDATA_REQUEST_EXECCOMMAND      = 0;
	SERVERDATA_REQUEST_AUTH             = 1;
	SERVERDATA_REQUEST_SEND_CONSOLE_LOG = 2;
}

message request
{
	optional int32     messageID   = 1;
	optional int32     messageType = 2;
    optional request_t requestType = 3;
    optional string    requestMsg  = 4;
    optional string    requestVal  = 5;
}
